doc/serial-timeout-troubleshoot.md

```diff
@@ -0,0 +1,80 @@
+# Troubleshooting "serial timeout" errors
+
+Several nRF Connect Desktop applications use serial ports to communicate with
+development kits and dongles. nRF Connect Desktop also uses serial ports to
+perform DFU (Device Firmware Update) when a devkit/dongle is running a 
+[DFU bootloader](http://infocenter.nordicsemi.com/topic/com.nordic.infocenter.sdk5.v15.0.0/sdk_app_serial_dfu_bootloader.html?cp=4_0_0_4_3_4).
+
+Serial ports might be [UARTs](http://infocenter.nordicsemi.com/topic/com.nordic.infocenter.nrf52840.ps/uart.html?cp=2_0_0_5_32)
+on the devkits connected to a RS-232 port on a computer,
+or they might be emulated through USB CDC ACM (also known as "Virtual COM port",
+"serial over USB" or other names). The later happens when using 
+[USB ACM CDC on the IMCU](http://infocenter.nordicsemi.com/topic/com.nordic.infocenter.nrf52/dita/nrf52/development/preview_dev_kit/vir_com_port.html?cp=2_1_5_4_1),
```

Comment:
Is 'USB ACM CDC' correct or was this meant to be 'USB CDC ACM'?

Reply:
I wonder as well, if the order matters.

Reply:
It's supposed to be USB CDC ACM (ACM is a subset of CDC). Will fix.